kubera-enterprise/values.yaml

```diff
@@ -24,6 +29,7 @@ mongodb:
         tag: 4.4.1-debian-10-r13
 
 ingress-nginx:
+    enablemodule: true
```

Comment:
Since you are using same key for both mongo and ingress, while referring, should we prepend .mongo and .ingress-nginx?

Reply:
`mongodb` and `ingress-nginx` we are using as sub chart in mongodb chart name is `mongodb` in ingress chart name is `ingress-nginx` and these 2 charts has it's values.yaml
To replace a child values.yaml we need to write  chield-chart-name.xyz -> xyz present in values.yaml of child.
On your question Yes if we want to use these values in our chart then we need to use like .mongo and .ingress-nginx. If we are using it child chart then we don't need .mongo or .ingress-nginx